docassemble/CivilActionCoverSheet/data/questions/Civil_Action_Cover_Sheet.yml

```diff
@@ -257,10 +257,12 @@ id: contract claims
 question: |
   Does this action involve contract claims?
 subquestion: |
-  If this civil action involves contract claims, provide a description of those claims and the total dollar value of the claims. Otherwise, skip this question. 
+  If this civil action involves contract claims, provide a description of those claims and the total dollar value of the claims. All expenses should be rounded to the nearest dollar. 
+  Otherwise, skip this question. 
```

Comment:
I don't think it's necessary to include this line because the user would have selected contract claims on the screen `id: questions pertaining to case`.